src/sentry/lang/native/symbolicator.py

```diff
@@ -28,18 +30,31 @@ def _task_id_cache_key_for_event(project_id, event_id):
     return f"symbolicator:{event_id}:{project_id}"
 
 
+@dataclass(frozen=True)
+class SymbolicatorTaskKind:
+    is_js: bool = False
+    is_low_priority: bool = False
+    is_reprocessing: bool = False
+
+    def with_low_priority(self, is_low_priority: bool) -> "SymbolicatorTaskKind":
+        return dataclasses.replace(self, is_low_priority=is_low_priority)
+
+
 class Symbolicator:
-    def __init__(self, project, event_id):
-        symbolicator_options = options.get("symbolicator.options")
-        base_url = symbolicator_options["url"].rstrip("/")
+    def __init__(self, task_kind: SymbolicatorTaskKind, project: Project, event_id: str):
+        URLS = settings.SYMBOLICATOR_POOL_URLS
+        pool = "default"
```

Comment:
nit: Preferably we have the string names of pools as constants (or enum) here and in settings.

Reply:
I defined them in `symbolicator.py`, defining that in the settings seems weird, and does not play nice with typing either. Importing that into the settings would also introduce circular dependencies I guess, so I just added a comment there.